tests/components/wled/conftest.py

```diff
@@ -73,3 +73,13 @@ async def init_integration(
     await hass.async_block_till_done()
 
     return mock_config_entry
+
+
+@pytest.fixture
+def enable_all_entities() -> Generator[AsyncMock, None, None]:
+    """Test fixture that ensures all entities are enabled in the registry."""
+    with patch(
+        "homeassistant.helpers.entity.Entity.entity_registry_enabled_default",
+        return_value=True,
+    ) as mock_entity_registry_enabled_by_default:
+        yield mock_entity_registry_enabled_by_default
```

Comment:
Maybe this would be useful in the top-level `conftest`?

Reply:
🤷  If more usage would be there, I think so?

Reply:
Done in [`7c80a0a` (#68087)](https://github.com/home-assistant/core/pull/68087/commits/7c80a0a3fa1db0b92f69e7c5f921bf141b87af79)

Reply:
I've had this question come up in reviews a few times.  It will be good to have a more generic solution.  Thanks 👍